x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -51,151 +50,137 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) =>
+                <EuiToolTip
+                  position="top"
+                  content={i18n.translate(
+                    'xpack.enterpriseSearch.inferencePipelineCard.modelState.notDeployed.fixLink',
+                    { defaultMessage: 'Fix issue in Trained Models' }
+                  )}
+                >
+                  <EuiButtonIcon
+                    aria-label={i18n.translate(
+                      'xpack.enterpriseSearch.inferencePipelineCard.modelState.notDeployed.fixLink',
+                      {
+                        defaultMessage: 'Fix issue in Trained Models',
+                      }
+                    )}
+                    data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-inferencePipeline-fixIssueInTrainedModels`}
+                    href={http.basePath.prepend(ML_MANAGE_TRAINED_MODELS_PATH)}
+                    display="base"
+                    size="xs"
+                    iconType="wrench"
                   />
-                </EuiFlexItem>
-                {pipeline.modelState === TrainedModelState.NotDeployed && (
-                  <EuiFlexItem grow={false} style={{ paddingRight: '1rem' }}>
-                    <EuiToolTip
-                      position="top"
-                      content={i18n.translate(
-                        'xpack.enterpriseSearch.inferencePipelineCard.modelState.notDeployed.fixLink',
-                        { defaultMessage: 'Fix issue in Trained Models' }
-                      )}
-                    >
-                      <EuiButtonIcon
-                        aria-label={i18n.translate(
-                          'xpack.enterpriseSearch.inferencePipelineCard.modelState.notDeployed.fixLink',
-                          {
-                            defaultMessage: 'Fix issue in Trained Models',
-                          }
-                        )}
-                        data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-inferencePipeline-fixIssueInTrainedModels`}
-                        href={http.basePath.prepend(ML_MANAGE_TRAINED_MODELS_PATH)}
-                        display="base"
-                        size="xs"
-                        iconType="wrench"
-                      />
-                    </EuiToolTip>
-                  </EuiFlexItem>
-                )}
-                <EuiFlexItem grow={false}>
-                  <EuiFlexGroup gutterSize="xs">
-                    <EuiFlexItem>
-                      <span>
-                        <MLModelTypeBadge type={modelType} />
-                      </span>
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiFlexItem>
-          </EuiFlexGroup>
+                </EuiToolTip>
+              </EuiFlexItem>
+            )}
+            <EuiFlexGroup direction="column" gutterSize="none">
+              <EuiFlexItem>
+                <div>
```

Comment:
I wonder if this `div` is only needed to fix some styling issue. Normally we try to avoid mixing HTML elements with EUI components.
Since the task is just for reorganizing code, let's leave it as-is.